archive.py

```diff
@@ -230,10 +237,12 @@ def write_topic(stream_name, stream, topic_name):
     f.close()
     o = open_outfile(md_root / Path(sanitize_stream(stream_name, stream['id'])), Path(sanitize_topic(topic_name) + '.html'), 'w+')
     write_topic_header(o, stream_name, stream['id'], topic_name)
-    o.write('\n{% raw %}\n')
+    if jekyll_output:
+        o.write('\n{% raw %}\n')
     write_topic_body(messages, stream_name, stream['id'], topic_name, o)
-    o.write('\n{% endraw %}\n')
-    o.write('\n{% include ' + str(last_updated_file) + ' %}')
+    if jekyll_output:
```

Comment:
@Mark-Simulacrum  is this what you wanted?

Reply:
I would prefer that in non-Jekyll mode we literally include the contents instead of via an include statement. That does mean that all of the files will change on each run, but that's true if you render through jekyll to a static site anyway.